tests/lib/nm/bond_test.py

```diff
@@ -61,19 +61,12 @@ def test_is_bond_type_id(NM_mock):
     assert nm.bond.is_bond_type_id(type_id)
 
 
-@mock.patch.object(nm.bond.connection, "ConnectionProfile")
-def test_get_bond_info(con_profile_mock, dev_mock):
+@mock.patch.object(nm.bond, "_get_options")
```

Comment:
It is improper to mock a private.

Reply:
Any suggestions?

I don't like change to code only for test purpose.

Reply:
You can mock `iglob` and `open` and then even cover more ground. I think that `open` has a special helper as it is common.

Reply:
BTW, I am also good with dropping this unit test in favor of `nm/bond_test.py`. But in order to "say" that, it is work its own commit with the reasoning in its message.

Reply:
ping